src/components/Authentication/AuthenticationTabs.jsx

```diff
@@ -0,0 +1,38 @@
+import { Login, Register } from "./";
+import {useState} from "react";
+const AuthenticationTabs = () => {
+    const [activeTab, setActiveTab] = useState("Login");
+    const tabClickhandler = (activeTabName) => setActiveTab(activeTabName)
+    return (
+        <section className="authentication-tabs p-10 m-10">
+            <article className="horizontal-tabs-container flex-column flex-wrap flex-gap-1 b-radius-2 p-5">
+                <article
+                    className="horizontal-tabs-btn-container flex-row flex-grow-1 justify-content-center align-center">
+                    <button
+                        className={`tabs-btn p-5 flex-row flex-grow-1 justify-content-center align-center ${activeTab === "Login"? "horizontal-tabs-btn-active":""}`}
+                        onClick={() => tabClickhandler("Login")}
+                        >Login</button>
```

Comment:
This is a great way of reducing code in the form of tabs to display a component depending on what user clicks!